src/helpers/Context.jsx

```diff
@@ -0,0 +1,26 @@
+import { createContext, useState } from "react";
+import PropTypes from "prop-types";
+
+export const CustomContext = createContext();
```

Comment:
The name itself says nothing about what is inside. How will you name your second context then?

Reply:
ok